client/network/src/protocol/notifications/tests.rs

```diff
@@ -262,8 +262,8 @@ fn reconnect_after_disconnect() {
 		loop {
 			// Grab next event from services.
 			let event = {
-				let s1 = service1.next();
-				let s2 = service2.next();
+				let s1 = service1.select_next_some();
+				let s2 = service2.select_next_some();
```

Comment:
Did you change this because you encountered a panic with `next()`? Or did you change this "just in case"?

Reply:
No panic, just does not compile anymore. I did a quick read through some PRs and found `select_next_some()` in an example usage

Reply:
Weird that it doesn't compile, but why not.

Reply:
The `next()` call is fine, but it would have required me to match on the `Option<...>` in the `match` below. Thought this was more concise.